src/triplet_validator.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import csv
+
+sah_endpoint = "https://www.ebi.ac.uk/ena/sah/api/validate"
+
+
+def validate_triplet(triplet, ggbn_id):
+    params = {
+        "value": triplet,
+        "qualifier_type": "specimen_voucher"
+    }
+    r = requests.get(sah_endpoint, params).json()
```

Comment:
There might be also triple IDs without a catalog number. If such a triplet is queried, it can be identified as being valid by the SAH API because it is mistaken with a "triplet" consisting of an institution code and a catalogue number. Indeed it could be a "triplet" with an institution code and an invalid collection code. As I am not sure if we want to catch cases like this either inside or outside of this function, it could be a source for errors.

Reply:
Right, good point. I had some logic here that checked if collection id was present and produced a "doublet" instead of a triplet. Thought the triplet building should be in a separate place, though, stay tuned.